### DISCUSSION TITLE
Question about the rgb-std

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#114] [Q&A]
Hi there, I found the code in the rgb-std here:
<img width="576" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/7d68a252-3d6d-45f7-a75d-7ff8c46f0c36">
In the AltLayer1, there is only liquid network, so why no bitcoin?

**Comment by dr-orlovsky** [comment]
Since the Bitcoin is always present by default and `Liquid` is added only as an additional network next to it. `AltLayer1` is the structure listing "other than Bitcoin supported networks".

**Comment by dr-orlovsky** [comment]
Why did you close the question before an answer?
